app/(dashboard)/mailboxes/[mailbox_slug]/settings/team/deleteMemberDialog.tsx

```
@@ -0,0 +1,159 @@
+        const description = result.updatedImmediately ? undefined : "Please refresh the page to see the changes";
+
+        toast.success(title, { description });
+      } else {
+        await removeTeamMember({ id: assignedToId.id, mailboxSlug });
+
+        toast.success("Member removed from the Team");
+      }
+    } catch (error) {
+      setLoading(false);
+      toast.error("Something went wrong");
+      console.error(error);
```

ESLint

[error] Unexpected console statement.

(no-console)